vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_wNa16.py

```diff
@@ -21,6 +22,7 @@ class CompressedTensorsWNA16(CompressedTensorsScheme):
     def __init__(self,
                  strategy: str,
                  num_bits: int,
+                 act_order: bool,
                  group_size: Optional[int] = None):
```

Comment:
Nit: act_order should be after group_size, since you can't have it without group_size being set